app/views/compute_resources_vms/form/libvirt/_base.html.erb

```diff
@@ -8,8 +8,11 @@
 <% checked = params[:host] && params[:host][:compute_attributes] && params[:host][:compute_attributes][:start] || '1' %>
 <%= checkbox_f f, :start, { :checked => (checked == '1'), :help_inline => _("Power ON this machine"), :label => _('Start'), :label_size => "col-md-2"} if new_vm && controller_name != "compute_attributes" %>
 
-<% firmware_type = new_vm ? 'automatic' : compute_resource.firmware_type(f.object.firmware, f.object.secure_boot) %>
-<%= field(f, :firmware, :disabled => !new_vm, :label => _('Firmware'), :label_help => _("Choose 'Automatic' to set the firmware based on the host's PXE Loader. If no PXE Loader is selected, it defaults to BIOS."), :label_size => "col-md-2") do
+<%
+  firmware_type = compute_resource.firmware_type(f.object.firmware, f.object.secure_boot)
+  firmware_help_block = _("If Compute Profile is set to 'Automatic', the firmware selection below was already auto-selected based on the PXE Loader. Setting firmware to 'Automatic' will     match the detected firmware based on the PXE Loader unless manually changed.") if new_vm && controller_name != 'compute_attributes'
```

Comment:
```suggestion
  firmware_help_block = _("If firmware on the compute profile is set to 'Automatic', the firmware below has already been pre-selected based on the PXE loader. If you set firmware to 'Automatic', Foreman will base the firmware on the PXE loader unless you select it manually.") if new_vm && controller_name != 'compute_attributes'
```
I'm not sure if I understand it correctly but this is how I would rephrase it. Also below.

Reply:
Also, will this text appear above or below the form field?

Reply:
you are right. thank you. forget this.